core/src/main/scala/org/scalarelational/instruction/Query.scala

```diff
@@ -55,4 +59,43 @@ case class Query[Expressions, Result](expressions: Expressions,
   def async = table.datastore.async {
     result
   }
+
+  def to[R](implicit manifest: Manifest[R]) = {
```

Comment:
The only impact to `core` this currently has is defining a generic type for inserts, updates, etc.  What would you think about creating a MappedInsert wrapper that has the generic type instead so that it remains clean in `core` unless you can think of a use-case for the generic type outside of `Mapper`?